template.yml

```diff
@@ -262,6 +264,82 @@ Resources:
       Environment:
         Variables:
           AWS_LAMBDA_EXEC_WRAPPER: /opt/change_runtime_directory
+          OUTPUT_BUCKET_NAME: !Ref OutputBucket
+          GET_OUTPUT_URL: !Sub "https://${OutputDomain}"
+  OutputBucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketName: !If [IsDevCondition, !Sub "cdo-dev-${SubDomainName}-output", !Sub "cdo-${SubDomainName}-output"] 
+      CorsConfiguration:
+        CorsRules:
+          - AllowedMethods: [GET]
+            AllowedOrigins: ['*']
+            AllowedHeaders: ['*']
+      BucketEncryption:
+        ServerSideEncryptionConfiguration:
+          - ServerSideEncryptionByDefault:
+              SSEAlgorithm: 'AES256'
+      LifecycleConfiguration:
+        Rules:
+          - Id: ExpirationRule
+            Status: Enabled
+            ExpirationInDays: 1
```

Comment:
Just a question (non-blocking). Is it easy to set this even lower? I could see 6 hours (or even 4) being a reasonable amount of time to retain these.

Reply:
The options I saw were either setting a specific timestamp for expiration or setting expiration in days: [doc](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-s3-bucket-lifecycleconfig-rule.html), and per [this](https://aws.amazon.com/blogs/aws/amazon-s3-object-expiration/) it seems expiration rules are only evaluated once per day. I agree it could be shorter, if we can find a way to do that I'm all for it!